app/api/routes/resource_retrieval.py

```diff
@@ -42,7 +43,19 @@ def get_resources():
     updated_after = request.args.get('updated_after')
     free = request.args.get('free')
 
+    apikey = request.headers.get('x-apikey')
```

Comment:
I think this needs to be `g.auth_key` to work with users logged in from the front end. I ran into this problem #395

Reply:
I think in #395 it made sense to do `g.auth_key` since it was already set during `@authenticate` decorator. This endpoint doesn't have that decorator so `g.auth_key` would always result in a key exception since it was never defined.

Can you double-check what your set up was? like did you get the authentication token from the backend running on localhost and used that token to hit resources API also running on localhost?

Reply:
I'll admit I haven't ran it yet, just gave a quick look over the code and it jumped out at me. I'll let you know when I try it if it works, I just don't know how it would if you're only looking for the `x-apikey` header and not the authorization token

Reply:
I see. That is taken care of on line 48 where it will first attempt to get the key via x-apikey header or via authentication header. That's literally copy and paste from authenticate decorator.

I couldn't test myself cuz I don't have backend setup locally. Let me know if you run into any problem.

Reply:
Ok well sorry I didn't run through it before commenting 😅 I'll try to test it locally today if I get the chance but it's Halloween so I'm not sure if I will. In the next couple days for sure